### PR TITLE
Add impact_patterns field to the event object.

### PR DESCRIPTION
**Pull request by medhatphq**
Updated the Event model to have impact_patterns field.

### CODE REVIEW

File: predicthq/endpoints/v1/events/schemas.py [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
+    ModelType,
+    PaginatedMixin,
+    Place,
     PolyModelType,
+    ResultSet,
+    ResultType,
+    SortableMixin,
+    StringListType,
+    StringModelType,
+    StringType,
```
Comment (medhatphq) [inline predicthq/endpoints/v1/events/schemas.py:27]: Sorry, this is just sorting imports by black. I should have done that as a separate commit.

File: predicthq/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "1.9.4"
+__version__ = "1.10.0"
```
Comment (medhatphq) [inline predicthq/version.py:1]: Wasn't too sure if I should bump the version here in my branch or after the merge to master? Also the tag should be created after the merge to master right?

**Review by chloext — APPROVED** [review]
looks good after updating the token
